src/Humanizer.Tests.Shared/InflectorTests.cs

```diff
@@ -280,6 +280,7 @@ public IEnumerator<object[]> GetEnumerator()
             yield return new object[] { "alumna", "alumnae" };
             yield return new object[] { "alumnus", "alumni" };
             yield return new object[] { "fungus", "fungi" };
+
```

Comment:
Break here because this is where the [original file](https://github.com/srkirkland/Inflector/blob/master/Inflector.Tests/PluralizeTests.cs) stopped.